container_service_extension/rde/models/common_models.py

```diff
@@ -44,6 +45,7 @@ def get_id(self):
             return self.id
 
 
+@dataclass_json(undefined=Undefined.EXCLUDE)
 @dataclass(frozen=True)
 class DefEntityType:
     """Represents the schema for Defined Entities."""
```

Comment:
interfaces: list[str]*

Reply:
Instead of creating another subclass, we can use 

from typing import Optional

@dataclass_json(undefined=Undefined.EXCLUDE)
@dataclass(frozen=True)
class DefEntityType:
  ...
  hooks: Optional[dict]
  ...